docker-compose.yml

```diff
@@ -1,37 +1,51 @@
 ---
-version: "3.7"
-
-# This docker-compose file is used to build and test the container
-
-secrets:
-  quote_txt:
-    file: ./src/secrets/quote.txt
-
+version: '3'
 services:
-  example:
-    # Run the container normally
+
+  li-pca-api:
     build:
-      # VERSION must be specified on the command line:
-      # e.g., --build-arg VERSION=0.0.1
       context: .
-      dockerfile: Dockerfile
-    image: cisagov/example
-    init: true
-    restart: "no"
-    environment:
-      - ECHO_MESSAGE=Hello World from docker-compose!
+      dockerfile: ./docker/api/Dockerfile
+    stdin_open: true
+    tty: true
+    container_name: li-pca-api
+    hostname: li-pca-api
+    restart: on-failure
+    links:
+      - mongodb
+    volumes:
+      - .:/usr/src/api
+    networks:
+      - backend
     ports:
-      - target: 8080
-        published: 8080
-        protocol: tcp
-        mode: host
-    secrets:
-      - source: quote_txt
-        target: quote.txt
+      - 8080:8080
+      - 8043:443
+      - 5678:5678
 
-  example-version:
+  li-pca-api-version:
     # Run the container to collect version information
-    image: cisagov/example
+    build:
+      context: .
+      dockerfile: ./docker/api/Dockerfile
     init: true
     restart: "no"
     command: --version
+
+  mongodb:
+    container_name: li-pca-db
+    hostname: li-pca-db
+    image: "mongo:3.6"
+    ports:
+      - "27017:27017"
+    command: --profile=1 --slowms=1
+    volumes:
+      - "mongodb:/data/db"
```

Comment:
Please use the [long syntax](https://docs.docker.com/compose/compose-file/compose-file-v3/#long-syntax-3).